src/Controller/SiteController.php

```diff
@@ -24,4 +31,72 @@ public function index(ServerRequestInterface $request): ResponseInterface
             ]
         );
     }
+
+    public function about(ServerRequestInterface $request): ResponseInterface
+    {
+        return $this->render(
+            'about',
+            [
+                'csrf' => $request->getAttribute(Csrf::REQUEST_NAME)
```

Comment:
I wonder if there's a way not to pass it around...

Reply:
btw., in this case it's not needed since it's GET-only.

Reply:
done.